openedx/core/djangoapps/theming/startup.py

```diff
@@ -2,13 +2,9 @@
 Startup code for Comprehensive Theming
 """
 
-from path import Path as path
-from django.conf import settings
-
-from .core import enable_comprehensive_theme
+from .core import try_enable_theme
 
 
 def run():
     """Enable comprehensive theming, if we should."""
-    if settings.COMP_THEME_DIR:
-        enable_comprehensive_theme(theme_dir=path(settings.COMP_THEME_DIR))
+    try_enable_theme()
```

Comment:
The `enable` method should know well enough to not actually enable itself if it hasn't been configured yet.